includes/class-wp-smime.php

```diff
@@ -111,7 +111,19 @@ public static function encrypt ( $message, $headers, $certificates ) {
         $outfile = $infile . '.enc';
 
         $plaintext  = ( is_array( $headers ) ) ? implode( "\n", $headers ) : $headers;
-        $plaintext .= "\n\n" . $message;
+
+	    // add content type and UTF-8 charset if not otherwise specified or missing
+	    if ( stripos( $plaintext, "Content-Type" ) === false ) {
```

Comment:
Please use Yoda conditions in accordance with the WordPress Core PHP style guidelines. Constants (like `false`) on the left side of the comparison operator.